README.md

```diff
@@ -93,32 +93,32 @@ Requirements:
 Compile and install:
 
 ```bash
-$ mvn install -DskipTests
+$ ./mvnw install -DskipTests
```

Comment:
does this script works for Windows environment?

Reply:
On Windows it's the mvn.cmd script that shall be used.

Reply:
How do people know to use `mvn.cmd`?

If people use its local maven installation, `mvn install ...` works across different operating systems. 

I am not against adding the mvn wrapper. But I don't think we should change the documentation. Once you change the documentation, you make people using windows hard to get started on building pulsar.

Reply:
you can still use class "mvn" command.
you are not forced to use mvnw.

IIRC on Windows when you have "mvnw.cmd" you can use "mvnw" as command, like for .bat files

so the usage is the same on Linux/Mac/Windows, it is always "./mvnw".  (probably on windows you can also omit './')

Reply:
Is someone able to test it on Windows ?

Reply:
On Windows, `./mvnw` won't work on standard terminal. You need to either use `mvnw` or `.\mvnw`